src/types.d.ts

```diff
@@ -8,12 +8,29 @@ type Primitive = string | boolean | number
 // configs & constructors
 // TODO: clean up this section when "constructor typing" work is
 //       being done
-interface ITransactionLimits {
-  config
-  level: number
-  onUsLimit: () => number
-  withdrawalLimit: () => number
-  oldEnoughForWithdrawalLimit: () => number
+
+type GenericLimits = {
+  oldEnoughForWithdrawalHours: number
+  oldEnoughForWithdrawalMicroseconds: number
+}
+
+type FeeRates = {
+  depositFeeVariable: number
+  depositFeeFixed: number
+  withdrawFeeVariable: number
+  withdrawFeeFixed: number
+}
+
+type UserLimitsArgs = { level: number; limitsConfig? }
+
+interface IUserLimits {
+  onUsLimit: number
+  withdrawalLimit: number
+}
+
+interface ITransactionLimits extends IUserLimits {
+  oldEnoughForWithdrawalLimit: number
```

Comment:
If we're adding the time units as suffixes on some places (like in the GenericLimit types), we should add it here as well to avoid confusion.

Reply:
**_Done: https://github.com/GaloyMoney/galoy/pull/393/commits/b01e2283937c65b6fe45bf22442a97c6ba3c466d_**